data/json/items/gun/38.json

```diff
@@ -238,8 +236,7 @@
         "ammo_restriction": { "357mag": 7, "38": 7 },
         "allowed_speedloaders": [ "38_speedloader" ]
       }
-    ],
-    "melee_damage": { "bash": 9 }
```

Comment:
Why are you removing the melee damage entry for this particular weapon? I agree that most of them make little sense and could do with auditing, but it seems a little strange that every other gun will have a melee damage capability save for this particular one.

Reply:
I implemented the Chiappa Rhino without one, I think it just makes sense to have melee damage value be defined automatically instead of manually set on firearms.

I admit that I did it "while I was at it" and therefore it may be considered out of scope.

Reply:
If the weapon gets melee damage assigned automatically, then I have no issue. I would simply have a problem if this was the singular gun that left enemies unharmed if you slugged them in the skull with it.

Reply:
I can't guarantee that the automatic melee damage is super well calculated mind you, it just seems a bit weird to be manually setting melee damage value on non melee weapons.

You think I should leave it as it is?